src/index.js

```diff
@@ -0,0 +1,234 @@
+import './style.css';
+import refresh from '../img/refresh.png';
+import enter from '../img/enter.png';
+import more from '../img/more.png';
+import trash from '../img/trash.png';
+import checkbox from '../modules/checkbox.js';
+
+const header = document.querySelector('.header');
+const addBar = document.querySelector('.add-bar');
+const list = document.querySelector('.list');
+const addInput = document.querySelector('.add-bar-text');
+
+const refreshIcon = new Image();
+const enterIcon = new Image();
+refreshIcon.src = refresh;
+enterIcon.src = enter;
+refreshIcon.classList.add('refresh-icon');
+enterIcon.classList.add('enter-icon');
+
+header.appendChild(refreshIcon);
+addBar.appendChild(enterIcon);
+
+let listArr = [];
+
+const listWrapperArr = [];
+const listContentArr = [];
+const listImgArr = [];
+const listMoreArr = [];
+const listTrashArr = [];
+const listCheckboxArr = [];
+const listTextArr = [];
+const taskInfo = [];
+
+function displayList() {
+  listArr.sort((x, y) => (
+    x.index - y.index
+  ));
+  for (let i = 0; i < listArr.length; i += 1) {
+    listWrapperArr[i] = document.createElement('li');
+    listWrapperArr[i].classList.add('list-wrapper');
+    list.appendChild(listWrapperArr[i]);
+
+    listContentArr[i] = document.createElement('div');
```

Comment:
Please don't repeat your self by repeating the same code, because it's not a good practice and its not professional